aqua/tagainijisho/Portfile

```diff
@@ -29,11 +29,28 @@ long_description    Tagaini Jisho is a free, open-source Japanese \
 homepage            http://www.tagaini.net/
 github.tarball_from releases
 
-checksums           rmd160  eda4b6c645ffce4298da6890260a246747bf3612 \
-                    sha256  6fe3c579ec1cea6c41393ac9a82dd12d0e240032d2bb7837a90e470f480aaf4e \
-                    size    24816175
+checksums           rmd160  83a9ecc7f1d5b5ba0ec6871c66a79e2590bb14e0 \
+                    sha256  10fa6aa3a2e0eea0f4456236ac3020db73c76f65cb31a76ad74a107b2d739003 \
+                    size    32366512
+
+# Needed for Qt5LinguistTools
+depends_lib-append  port:qt5-qttools
```

Comment:
there is specific way to declare qt5 dependencies, I think with "qt5.depends" but please check the PortGroup